core/src/cs/eng1/piazzapanic/customer/CustomerManager.java

```diff
@@ -188,7 +189,7 @@ public void act(float delta) {
     }
 
     public void checkSpawn(float delta) {
-        if (spawnTimer.tick(delta)) {
+        if (spawnTimer.tick(delta) && spawnedCustomers != totalCustomers) {
```

Comment:
```suggestion
        if (spawnedCustomers != totalCustomers && spawnTimer.tick(delta)) {
```
This would be slightly more efficient in scenario mode.